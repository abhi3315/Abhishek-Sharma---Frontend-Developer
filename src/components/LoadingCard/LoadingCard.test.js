import { render, cleanup } from '@testing-library/react';

import LoadingCard from './LoadingCard';

afterEach(cleanup);

describe('LoadingCard', () => {
	it('should render correctly', () => {
		const { container } = render(<LoadingCard />);
		expect(container).toMatchSnapshot();
	});
});
