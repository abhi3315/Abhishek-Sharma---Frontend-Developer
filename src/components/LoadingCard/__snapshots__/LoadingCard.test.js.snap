// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingCard should render correctly 1`] = `
<div>
  <div
    class="max-w-sm p-4 border border-gray-200 rounded shadow animate-pulse md:p-6 dark:border-gray-700 w-full mx-auto"
    role="status"
  >
    <div
      class="h-2.5 bg-gray-200 rounded-full dark:bg-gray-700 w-48 mb-4"
    />
    <div
      class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 mb-2.5"
    />
    <div
      class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 mb-2.5"
    />
    <div
      class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 mb-2.5"
    />
    <div
      class="h-2.5 bg-gray-200 rounded-full dark:bg-gray-700 w-48 mb-2.5"
    />
    <div
      class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 mb-2.5"
    />
    <div
      class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 mb-2.5"
    />
    <div
      class="h-2 bg-gray-200 rounded-full dark:bg-gray-700"
    />
    <span
      class="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
